src/status_im/contexts/wallet/events.cljs

```diff
@@ -179,9 +179,23 @@
      {:fx [[:dispatch [:wallet/create-derived-addresses account-details on-success]]]})))
 
 (rf/reg-event-fx :wallet/select-bridge-to
+ (fn [{:keys [db]} [{:keys [token stack-id]}]]
+   (let [to-address (get-in db [:wallet :current-viewing-account-address])]
+     {:db (->
+            db
+            (assoc-in [:wallet :ui :send :token] token)
+            (assoc-in [:wallet :ui :send :to-address] to-address))
+      :fx [[:navigate-to-within-stack [:wallet-bridge-to stack-id]]]})))
```

Comment:
We should dispatch `:navigate-to-within-stack` event here

Reply:
e.g 
` :fx [[:dispatch [:navigate-to-within-stack [:wallet-bridge-to stack-id]]]]`